Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -185,7 +185,7 @@ public class PBXProjGenerator {
             for (platform, files) in carthageFrameworksByPlatform {
                 let platformGroup: PBXGroup = addObject(
                     PBXGroup(
-                        children: files.sorted { $0.nameOrPath < $1.nameOrPath },
+                        children: files.sorted(by: PBXFileElement.areNamePathInIncreasingOrder(_:_:)),
```

Comment:
I don't think we actually don't need to sort this at all as the files are sorted later
```suggestion
                        children: Array(files),
```